src/utils/index.ts

```diff
@@ -0,0 +1,18 @@
+declare global {
+    interface KopiaUI {
+        selectDirectory(callback: (selectedPath: string) => void): void
+        browseDirectory(path: string): void
+    }
+
+    interface Window {
+        kopiaUI?: KopiaUI;
+    }
+}
+
+export * from './FsSize';
```

Comment:
do we need to re-export those things? can we instead fix all usages to import from the right place?

Reply:
I plan to use `import { FsSize } from '@kopia/utils'` because I think it simplifies renaming and adding files in the future and is much more readable. For that we need to re-export them from an `index.ts` file as far as I am concerned.